packages/core/src/Stores/traders-hub-store.js

```diff
@@ -470,27 +471,10 @@ export default class TradersHubStore extends BaseStore {
     }
 
     getAvailableMt5Accounts() {
-        if (this.is_eu_user && !this.is_demo_low_risk) {
-            this.available_mt5_accounts = this.available_cfd_accounts.filter(account =>
-                ['EU', 'All'].some(region => region === account.availability)
-            );
-            return;
-        }
-
-        if (this.financial_restricted_countries) {
-            this.available_mt5_accounts = this.available_cfd_accounts.filter(
-                account => account.market_type === 'financial' && account.platform === CFD_PLATFORMS.MT5
-            );
-            return;
-        }
-
-        if (this.CFDs_restricted_countries) {
-            this.available_mt5_accounts = this.available_cfd_accounts.filter(
-                account =>
-                    account.market_type !== 'financial' &&
-                    account.market_type !== 'all' &&
-                    account.platform === CFD_PLATFORMS.MT5
-            );
+        if (this.root_store.client.is_logged_in) {
+            this.available_mt5_accounts = this.available_cfd_accounts.filter(account => {
+                return account.platform === CFD_PLATFORMS.MT5 && this.available_accounts[account.product];
+            });
```

Comment:
Not sure why we are still doing this. We are also doing the same on lines 481-483. Can you please check?